File: PYTHON/modo.py
modo = input("Ingrese el modo (A-automatico, M-manual, S-seguro): ")

if modo.upper() == 'A':
    print("Modo automatico activado.")
elif modo.upper() == 'M':
    print("Modo manual activado.")
elif modo.upper() == 'S':
    print("Modo seguro activado.")
else:
    print("Error: Modo invalido.")

File: C/modo.c
#include <stdio.h>

int main() {
    char modo;

    printf("Ingrese el modo (A-automatico, M-manual, S-seguro): ");
    scanf(" %c", &modo);  // El espacio antes de %c evita problemas con el buffer

    if (modo == 'A' || modo == 'a') {
        printf("Modo automatico activado.\n");
    } 
    else if (modo == 'M' || modo == 'm') {
        printf("Modo manual activado.\n");
    } 
    else if (modo == 'S' || modo == 's') {
        printf("Modo seguro activado.\n");
    } 
    else {
        printf("Error: Modo invalido.\n");
    }

    return 0;
}
